ajax/libs/downloadjs/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/rndme/download.js"
+    "url": "https://github.com/rndme/download"
```

Comment:
@sufuf3 I just want to be sure I understand.  You want me to add an autoupdate section to skel-layers like you are showing me for downloadjs ?
_